modules/sandboxed-containers-deleting-namespace-web-console.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-namespace-web-console_{context}"]
+= Deleting a namespace using the web console
+
+You can delete a namespace by using the {product-title} web console.
+[NOTE]
+====
+If you do not have permissions to delete the namespace, the *Delete Namespace* option is not available.
+====
+
+.Prerequisites
+
+* You have {product-title} 1.0.0 installed.
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+.Procedure
+
+. Navigate to *Administration* → *Namespaces*.
+. Locate the namespace that you want to delete in the list of namespaces.
+. On the far right side of the namespace listing, select *Delete Namespace* from the *Options* menu .
```

Comment:
s/far right side/rightmost side/

ISG: 
**rightmost** _adj_
Use to refer to something farthest to the right (for example, on a page or a panel).